CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Updated e2e tests (`refund` and `inverted_refund`) to new e2e test format.
```

Comment:
Do our users care that much that we need a changelog entry? :grimacing:

Reply:
hehe... mergify complained so I was a good boy and updated it :D

Reply:
The "complaint" is more a reminder that you should think about whether it needs updating and not that you should update it, regardless of what the change is.

I don't think we need to bother users with internal changes like this.